gensim/similarity_learning/DRMM_TKS_Model.py

```diff
@@ -0,0 +1,468 @@
+import logging
```

Comment:
one more DRMM_TKS? I don't catch

Reply:
`DRMM_TKS_Model.py` has the easy user interface where user enters a list of words 
`DRMM_TKS.py` in `gensim.models.experimental` is the actual model definition without any assumptions.

Reply:
Why this "split", any reasons?

Reply:
I thought it would be better to have the functionality of
1. Keras Model AND
2. list of words preprocessing to be done separately.

I will move the keras model class to the preprocessing file like word2vec.py has.er
so there'll be only one file
@menshikh-iv

Reply:
@aneesh-joshi where you see "preprocessing" in word2vec.py (or what do you mean by "preprocessing")?

Reply:
by preprocessing, I meant stuff like `build_vocab`, etc which doesn't much do with the w2v Neural Net training.
Anyway, I have moved it all to one file.